indi-toupbase/indi_toupbase.h

```diff
@@ -453,6 +455,15 @@ class ToupBase : public INDI::CCD
         //#############################################################################
         // Properties
         //#############################################################################
+        ISwitchVectorProperty BinningModeSP;
+        ISwitch BinningModeS[2];
+        enum
+        {
+            TC_BINNING_AVG,
+            TC_BINNING_ADD,
+        };
```

Comment:
Use either this enum or the BINNING_MODE one defined below. Having 2 is not necessary.